Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/familytest/core/storage/FamilyTestDatabaseTest.kt

```diff
@@ -113,7 +113,7 @@ class FamilyTestDatabaseTest : BaseTestInstrumentation() {
         dao.getAllActive().first().size shouldBe 1
         val entries = dao.getAllInRecycleBin().first()
         entries.size shouldBe 1
-        entries[0]!!.test.coronaTest.state shouldBe CoronaTest.State.NEGATIVE
+        entries[0]!!.test.coronaTest.state shouldBe CoronaTest.State.RECYCLED
```

Comment:
how  is this  relevant to Json Refactoring?

Reply:
Fixed test and changed back to negative